scripts/release_scripts/initial_release_prep.py

```diff
@@ -192,24 +170,13 @@ def main():
         release_constants.ONE_TIME_JOBS_SPREADSHEET_URL)
     common.ask_user_to_confirm(
         'Please copy the names of the jobs to be run for this release along '
-        'with author names, author mail ids & instruction docs.')
+        'with author names, author mail ids & instruction docs.\n'
+        'Note: Copy only those jobs that have been successfully run '
+        'on backup server.')
     common.open_new_tab_in_browser_if_possible(
         release_constants.RELEASE_DRIVE_URL)
     common.ask_user_to_confirm(
         'Please enter the above job names to release journal.')
-    python_utils.PRINT(get_mail_message_template())
```

Comment:
Just to double-check -- why these deletions?

Reply:
We are changing the process as discussed in the mail thread. The author of a PR who adds a new job will need to run the job on backup server before getting the PR merged. So, the criteria to run jobs on backup server before release can be removed.